### PR TITLE
Inline UoF infographics and Icons into Fundraising Banners

### PR DESCRIPTION
**Pull request by Sperling-0**
- Add Infografics and Icon svgs as Vue components
    
    - No banners should include images that are coming from `fundraising-frontend-content` repo
    - Update content repo

- Remove unused Icon items
-  Use IconClose component instead of the svg 
- Introduce slot for infographic in FundsContent component 
- Pass required infographic component as a slot in Banner component files 
- Center UoF org chart horizontally
- Use swatch for close icon

Ticket: https://phabricator.wikimedia.org/T375848

### CODE REVIEW

**Review by gbirke — APPROVED** [review]

**Comment by moiikana** [comment]
on previous banners (e.g. https://de.wikipedia.org/?banner=B24_WMDE_Desktop_DE_04_ctrl&devMode) the use of funds graphics (info graphic and close icon) looked like this (on a mobile screen size):
Bildschirmfoto vom 2024-10-10 13-03-06


currently in this PR it looks like this:
Bildschirmfoto vom 2024-10-10 13-03-18
1) there used to be some more centering/spacing for the info graphic before, but the centering of it could overall be improved in this PR
2) the close icon needs a white background, so it's still well readable when it's floating over text

**Review by moiikana — CHANGES_REQUESTED** [review]
(see comment above)

File: src/themes/UseOfFunds/FundsContent.scss [hunk 1]
```diff
@@ -132,33 +132,13 @@ $padding-mobile: 16px;
...
-		}
 	}
 
 	&-orgchart-text {
 		flex: 1 1 auto;
 	}
-	&-orgchart-image img {
-		padding: 0 50px;
+	&-orgchart-image {
+		text-align: center;
```
Comment (Abban) [inline src/themes/UseOfFunds/FundsContent.scss:141]: We need to keep the padding here:

```suggestion
		text-align: center;
		padding: 0 50px;
```

File: src/components/UseOfFunds/Icons/IconClose.vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<template>
+	<svg width="32" height="32" viewBox="0 0 32 32" fill="#ffffff" xmlns="http://www.w3.org/2000/svg">
```
Comment (Abban) [inline src/components/UseOfFunds/Icons/IconClose.vue]: This needs to respect dark mode so we remove the fill and make sure it has a background in css using my suggestions in the main comment:

```suggestion
	<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

**Review by Abban — CHANGES_REQUESTED** [review]
It's nearly there.

I spotted an issue with the swatching, where the css variable for the close button background isn't being properly set. You need to make a couple of changes to fix it:

1. Change line 2 in `src/components/UseOfFunds/FundsModal.vue` to

```html
<div :class="[ 'banner-modal funds-modal', { 'is-visible': isFundsModalVisible } ]">
```

2. Cange the selectors in `src/themes/UseOfFunds/swatches/skin_default.scss` and `src/themes/UseOfFunds/swatches/skin_vector-2022.scss` from `.use-of-funds` to `.funds-modal`.

**Review by Abban — APPROVED** [review]

**Comment by moiikana** [comment]
close button looks good now :+1: 
while checking I noticed that 
- on the **desktop fallback banner** (<- only appears on very small screens on (refresh the page))
Bildschirmfoto vom 2024-10-17 17-48-04
- and on the **de mobile banner**
the infographic in the UoF is missing.

**Comment by Sperling-0** [comment]
> close button looks good now 👍 while checking I noticed that
> 
>     * on the **desktop fallback banner** (<- only appears on very small screens on (refresh the page))
>       Bildschirmfoto vom 2024-10-17 17-48-04
> 
>     * and on the **de mobile banner**
>       the infographic in the UoF is missing.

Now, these issues have been fixed. I re-based this PR from/to main.

**Review by moiikana — APPROVED** [review]

**Pull request merged**